### DISCUSSION TITLE
Request for Additional Model Support in GPT Computer Assistant

### DISCUSSION THREAD

**Discussion by Envertix** [crewAIInc/crewAI#1309] [Q&A]
I’ve been exploring GPT Computer Assistant, and while it provides support for models like Llama 3 and LLaVA, I’d love to see more options available. Specifically, I’m interested in using the Llama 3.1 Storm 8B model from Hugging Face (https://huggingface.co/akjindal53244/Llama-3.1-Storm-8B). Is there any plan to expand model support or allow users to integrate custom models? It would be great to have more flexibility with model selection!

**Comment by theCyberTech** [comment]
What is  GPT Computer Assistant?
